src/background/registries/nearRegistry.ts

```diff
@@ -80,8 +80,10 @@ export class NearRegistry implements Registry {
         'transferOwnership',
         'createModule',
         'addContextId',
+        'addAdmin',
         'addModuleWithContexts',
         'removeContextId',
+        'removeAdmin',
```

Comment:
Remove it. The NEAR smart-contract of dapplet registry doesn't contain admin functionality.